### PR TITLE
Update to 2.42.8

### PR DESCRIPTION
**Pull request by cbouss**
Repo: https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/tree/2.42.8
Changelog: https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/2.42.8/NEWS
Changes to previous: https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/compare/2.42.6...2.42.8?from_project_id=1548
License: https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/2.42.8/COPYING
Build files:
- https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/2.42.8/meson.build
- https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/2.42.8/meson_options.txt

Jira: https://anaconda.atlassian.net/browse/DSNC-4913

Changes:
- Update version to 2.42.8
- Update about section
- Update build scripts
- Enabled windows build

Todo: remove conda_build_config.yaml and abs.yaml before merging

### CODE REVIEW

**Review by ScottBailey — APPROVED** [review]
This looks good to me.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,59 +1,56 @@
...
-    - jpeg
+    - jpeg    # [win]
+    - libffi  # [win]
 
 requirements:
   build:
     - pkg-config
     - {{ compiler('c') }}
-    # See https://github.com/conda-forge/atk-feedstock/pull/14#issuecomment-610061908
     - meson ==0.56.2  # [osx and arm64]
```
Comment (varlackc) [inline recipe/meta.yaml:29]: The only version of `meson` that seems to be specified is `0.55.3`

https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/blob/2.42.8/meson.build#L9

We might have to investigate if we still should have a different version for `osx` and `arm64`

https://github.com/AnacondaRecipes/gdk-pixbuf-feedstock/blob/cb_2.42.8/recipe/meta.yaml#L29

**Review by varlackc — COMMENTED** [review]
we might have to investigate on the different dependencies like `meson` that have different pinnings for different architectures.

**Review by skupr-anaconda — APPROVED** [review]

**Review by tobijk — APPROVED** [review]
LGTM.

I'm not sure if you are aware of it, though: when you set a constraint like

```
libffi >=3.3
```

This will override any pinning set for libffi in cbc.yml (and may disable any constraints that may have been specified in libffi's run_exports section, though I'm not certain about this one anymore, please check).

I.e. you might be removing an upper or lower bound determined by the library's maintainer without noticing. And you might be linking against something completely different from what cbc.yaml mandates.

**Pull request merged**